public/app/plugins/panel/graph/module.ts

```diff
@@ -319,8 +319,9 @@ class GraphCtrl extends MetricsPanelCtrl {
   exportCsv() {
     var scope = this.$scope.$new(true);
     scope.seriesList = this.seriesList;
+    scope.isUtc = this.dashboard.isTimezoneUtc();
     this.publishAppEvent('show-modal', {
-      templateHtml: '<export-data-modal data="seriesList"></export-data-modal>',
+      templateHtml: '<export-data-modal data="seriesList" utc="isUtc"></export-data-modal>',
```

Comment:
you do not need to send this flag like this, the ExportDataModalCtrl can inject the dashboardSrv and get the current dashboard and check timezone that way.